libraries/chain/asset_evaluator.cpp

```diff
@@ -299,6 +304,10 @@ void_result asset_update_evaluator::do_evaluate(const asset_update_operation& o)
               "Incorrect issuer for asset! (${o.issuer} != ${a.issuer})",
               ("o.issuer", o.issuer)("a.issuer", a.issuer) );
 
+   FC_ASSERT( d.head_block_time() >= HARDFORK_CORE_BSIP87_TIME
+              || !o.new_options.extensions.value.force_settle_fee_percent.valid(),
+              "A BitAsset's FSFP cannot be set before Hardfork BSIP87" );
+
```

Comment:
Please add checks in `proposal_create_evaluator` as well.

Reply:
Added in cd4753b.